Tools/ardupilotwaf/ap_library.py

```diff
@@ -140,6 +140,7 @@ class ap_library_check_headers(Task.Task):
     dispatched_headers = set()
     whitelist = (
         'libraries/AP_Vehicle/AP_Vehicle_Type.h',
+        'libraries/AP_Scheduler/AP_Scheduler.h',
```

Comment:
The only reason that is needed is because SCHEDULER_DEFAULT_LOOP_RATE is dependent on vehicle and as it is in template, must be in .h file.
I do not like that exception either, but right now have no idea how to make it different.